jekyll/_cci2/troubleshooting.md

```diff
@@ -7,6 +7,42 @@ description: "Troubleshooting steps to take in CircleCI 2.0"
 ---
 This document describes an initial set of troubleshooting steps to take if you are having problems with your CircleCI 2.0 installation on your private server or cloud. **Note:** It is generally a good idea to review the support bundle to check for problems first before beginning troubleshooting steps.
 
+
+## Setting up ELB CERTS
+CircleCI requires a few more steps to get ELB certs working as your primary certs. The steps on how to accomplish this are below.
+
+**Note:** You'll only need HTTP below, if you plan on using HTTP requests, otherwise you can leave it out. 
+
+1.) You'll need to following ports on your ELB
+
+Load BalancerProtocol | Load Balancer Port | Instance Protocol | Instance Port | Cipher | SSL Certificate
+----------|----------|----------|----------|----------|----------
+SSL| 443 | SSL | 443 | Change | your-cert
+HTTP | 80 | HTTP | 80 | N/A | N/A
+HTTPS | 8800 | HTTPS | 8800| Change | your cert
+SSL | 8081 | SSL | 8081 | Change | your cert
+SSL | 3000 | SSL | 3000 | Change | your cert
+SSL|8082| SSL| 8082 | Change | your cert
```

Comment:
I forgot what 8082 is for, and might not be needed. It shouldn't harm anything though.